x/iscn/types/iscnmsg_test.go

```diff
@@ -0,0 +1,154 @@
 114, 115, 105, 118, 101, 32, 102, 117, 110, 99, 116, 105, 111, 110, 32, 119, 105, 116, 104, 32, 109, 97, 116, 114, 105, 120, 32, 109, 117, 108, 116, 105, 112, 108, 105, 99, 97, 116, 105, 111, 110, 46, 34, 44, 34, 107, 101, 121, 119, 111, 114, 100, 115, 34, 58, 34, 109, 97, 116, 114, 105, 120, 44, 114, 101, 99, 117, 114, 115, 105, 111, 110, 34, 44, 34, 116, 105, 116, 108, 101, 34, 58, 34, 228, 189, 191, 231, 148, 168, 231, 159, 169, 233, 153, 163, 232, 168, 136, 231, 174, 151, 233, 129, 158, 230, 173, 184, 233, 151, 156, 228, 191, 130, 229, 188, 143, 34, 44, 34, 117, 114, 108, 34, 58, 34, 104, 116, 116, 112, 115, 58, 47, 47, 110, 110, 107, 107, 101, 110, 46, 103, 105, 116, 104, 117, 98, 46, 105, 111, 47, 112, 111, 115, 116, 47, 114, 101, 99, 117, 114, 115, 105, 118, 101, 45, 114, 101, 108, 97, 116, 105, 111, 110, 47, 34, 44, 34, 117, 115, 97, 103, 101, 73, 110, 102, 111, 34, 58, 34, 104, 116, 116, 112, 115, 58, 47, 47, 99, 114, 101, 97, 116, 105, 118, 101, 99, 111, 109, 109, 111, 110, 115, 46, 111, 114, 103, 47, 108, 105, 99, 101, 110, 115, 101, 115, 47, 98, 121, 47, 52, 46, 48, 34, 44, 34, 118, 101, 114, 115, 105, 111, 110, 34, 58, 49, 125, 44, 34, 115, 116, 97, 107, 101, 104, 111, 108, 100, 101, 114, 115, 34, 58, 91, 123, 34, 99, 111, 110, 116, 114, 105, 98, 117, 116, 105, 111, 110, 84, 121, 112, 101, 34, 58, 34, 104, 116, 116, 112, 58, 47, 47, 115, 99, 104, 101, 109, 97, 46, 111, 114, 103, 47, 97, 117, 116, 104, 111, 114, 34, 44, 34, 101, 110, 116, 105, 116, 121, 34, 58, 123, 34, 64, 105, 100, 34, 58, 34, 100, 105, 100, 58, 99, 111, 115, 109, 111, 115, 58, 53, 115, 121, 50, 57, 114, 51, 55, 103, 102, 120, 118, 120, 122, 50, 49, 114, 104, 52, 114, 48, 107, 116, 112, 117, 99, 52, 54, 112, 122, 106, 114, 109, 122, 50, 57, 103, 52, 53, 34, 44, 34, 110, 97, 109, 101, 34, 58, 34, 67, 104, 117, 110, 103, 32, 87, 117, 34, 125, 44, 34, 114, 101, 119, 97, 114, 100, 80, 114, 111, 112, 111, 114, 116, 105, 111, 110, 34, 58, 57, 53, 125, 44, 123, 34, 99, 111, 110, 116, 114, 105, 98, 117, 116, 105, 111, 110, 84, 121, 112, 101, 34, 58, 34, 104, 116, 116, 112, 58, 47, 47, 115, 99, 104, 101, 109, 97, 46, 111, 114, 103, 47, 99, 105, 116, 97, 116, 105, 111, 110, 34, 44, 34, 100, 101, 115, 99, 114, 105, 112, 116, 105, 111, 110, 34, 58, 34, 84, 104, 101, 32, 98, 108, 111, 103, 32, 112, 111, 115, 116, 32, 114, 101, 102, 101, 114, 114, 101, 100, 32, 116, 104, 101, 32, 109, 97, 116, 114, 105, 120, 32, 102, 111, 114, 109, 32, 111, 102, 32, 99, 111, 109, 112, 117, 116, 105, 110, 103, 32, 70, 105, 98, 111, 110, 97, 99, 99, 105, 32, 110, 117, 109, 98, 101, 114, 115, 46, 34, 44, 34, 102, 111, 111, 116, 112, 114, 105, 110, 116, 34, 58, 34, 104, 116, 116, 112, 115, 58, 47, 47, 101, 110, 46, 119, 105, 107, 105, 112, 101, 100, 105, 97, 46, 111, 114, 103, 47, 119, 105, 107, 105, 47, 70, 105, 98, 111, 110, 97, 99, 99, 105, 95, 110, 117, 109, 98, 101, 114, 34, 44, 34, 114, 101, 119, 97, 114, 100, 80, 114, 111, 112, 111, 114, 116, 105, 111, 110, 34, 58, 53, 125, 93, 125, 125, 125,
```

Comment:
Prefer presenting in string form so developers may know what this is:
```suggestion
	msgCreateIscnRecordBytesNoNonce = []byte(`{"type":"likecoin-chain/MsgCreateIscnRecord","value":{"from":"cosmos1y54exmx84cqtasvjnskf9f63djuuj68p7hqf47","record":{"contentFingerprints":["hash://sha256/9564b85669d5e96ac969dd0161b8475bbced9e5999c6ec598da718a3045d6f2e"],"contentMetadata":{"@context":"http://schema.org/","@type":"CreativeWorks","author":"https://github.com/nnkken","datePublished":"2019-04-19","description":"An article on computing recursive function with matrix multiplication.","keywords":"matrix,recursion","title":"使用矩陣計算遞歸關係式","url":"https://nnkken.github.io/post/recursive-relation/","usageInfo":"https://creativecommons.org/licenses/by/4.0","version":1},"stakeholders":[{"contributionType":"http://schema.org/author","entity":{"@id":"did:cosmos:5sy29r37gfxvxz21rh4r0ktpuc46pzjrmz29g45","name":"Chung Wu"},"rewardProportion":95},{"contributionType":"http://schema.org/citation","description":"The blog post referred the matrix form of computing Fibonacci numbers.","footprint":"https://en.wikipedia.org/wiki/Fibonacci_number","rewardProportion":5}]}}}`)
	msgCreateIscnRecordBytesNonce1 = []byte(`{"type":"likecoin-chain/MsgCreateIscnRecord","value":{"from":"cosmos1y54exmx84cqtasvjnskf9f63djuuj68p7hqf47","nonce":"1","record":{"contentFingerprints":["hash://sha256/9564b85669d5e96ac969dd0161b8475bbced9e5999c6ec598da718a3045d6f2e"],"contentMetadata":{"@context":"http://schema.org/","@type":"CreativeWorks","author":"https://github.com/nnkken","datePublished":"2019-04-19","description":"An article on computing recursive function with matrix multiplication.","keywords":"matrix,recursion","title":"使用矩陣計算遞歸關係 式","url":"https://nnkken.github.io/post/recursive-relation/","usageInfo":"https://creativecommons.org/licenses/by/4.0","version":1},"stakeholders":[{"contributionType":"http://schema.org/author","entity":{"@id":"did:cosmos:5sy29r37gfxvxz21rh4r0ktpuc46pzjrmz29g45","name":"Chung Wu"},"rewardProportion":95},{"contributionType":"http://schema.org/citation","description":"The blog post referred the matrix form of computing Fibonacci numbers.","footprint":"https://en.wikipedia.org/wiki/Fibonacci_number","rewardProportion":5}]}}}
`)
```

Also, for the test cases of `GetSignBytes`, the record should be simplified, so developers may notice the difference in `nonce` field easier.

```Go
	createIscnRecord1 = IscnRecord{
		ContentMetadata: IscnInput(`{}`),
	}
```

(This is a unit test, so it should care less other stuff and only focus on the specific functionality tested by the test case, so the data should be simpler; in contrast, for a more integrated test, we would make the test data more complicated and closer to real life data)